lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -112,22 +112,13 @@ function Test-ProjectPackage {
   param ($Path, $Algorithm = 'SHA256', $Hash)
 
   if (-not (get-command get-filehash -ErrorAction 'SilentlyContinue')) {
```

Comment:
Don't we have to use our `Get-FileHash` so that we use our `Get-SHA256Converter`? 

The original code added `Get-FileHash` behind that if, so I figured some platforms were missing it. So I wouldn't remove that even if we don't need it.